### DISCUSSION TITLE
欢迎来讨论和分享你的学习体验

### DISCUSSION THREAD

**Discussion by shengxinjing** [shengxinjing/it-roadmap#3] [通知]
大家有问题都可以在这里或者去群里提问

一起好好学习 天天向上把   
山底太挤，我们山顶见

**Comment by chengjunjian2020** [comment]
1.如果是一门陌生技能，比如说去年年底入职一家公司react，之前我用的vue，没看过vue多少源码，但是使用还算熟悉，我看看项目代码，后面就是不断百度百度，用vue 思维写react，用的还是class风格的react
2.我学习不会学习不太用到的东西，比如说游戏引擎，我们公司视觉部门做的3D项目相关的，不太会学习因为前端没学完
3.比如学习一门组件到底值得不？用得到不比如说学习iScroll，前几天想学后来想到算了没必要花太大时间研究他的滚动算法
4.学习要有规划要又目的坚持
